src/index.ts

```diff
@@ -269,6 +269,16 @@ process.on("SIGUSR2", async () => await cleanupJobResources(jobs));
             description: "The name, without the architecture, of a gitlab hosted runner to emulate. See here: https://docs.gitlab.com/ee/ci/runners/hosted_runners/linux.html#machine-types-available-for-linux---x86-64",
             choices: GitlabRunnerPresetValues,
         })
+        .option("color", {
+            requiresArg: false,
+            default: false,
+            // placeholder for the --no-color option
+            hidden: true,
+        })
+        .option("no-color", {
+            requiresArg: false,
+            description: "Disable all color.",
+        })
```

Comment:
This is redundant
```suggestion
```

Reply:
hacky, but added that in for the --help message haha

not sure if there's a nicer way for it